OrbitLinuxTracing/PerfEvent.h

```diff
@@ -369,6 +369,17 @@ class TaskRenamePerfEvent : public TracepointPerfEvent {
   }
 };
 
+class TracepointEvent : public TracepointPerfEvent {
```

Comment:
The name of this class is too generic: what is the difference between `TracepointEvent` and `TracepointPerfEvent`? Actually, why do you even need this class? Couldn't you use `TracepointPerfEvent` directly and just add `GetPid` and `GetTid` to that?

Reply:
Thanks for the comment! One of the differences is that TracepointEvent does not inherit from anywhere and the ring buffer has the type perf_event_sample_id_tid_time_streamid_cpu instead of perf_event_raw_sample_fixed since only the contents of that structure is read. Also, TracepointEvent does not  need to have the field 'std::unique_ptr<uint8_t[]> tracepoint_data' since the data particular of every tracepoint is not read. I created TracepointEvent which is an independent class of TracepointPerfEvent because it had fields not necessary at this moment and it reserved additional memory that was not really put to use (such as space for the header, size and data[char]). However, I get what you mean with the name as being too generic and, if you think it would be a good idea, I could use TracepointPerfEvent even though I do not read all its contents at this point.

Reply:
Ah...I have just seen that this is an outdated comment. In the meantime, TracepointEvent does not depend on anything.